Tests/SwiftGenKitTests/TestsHelper.swift

```diff
@@ -187,11 +219,29 @@ class Fixtures {
     let path = self.path(for: name, subDirectory: "StencilContexts/\(sub.rawValue)")
 
     guard let data: String = try? path.read(),
-      let yaml = try? Yams.load(yaml: data, .basic),
+      let yaml = try? Yams.load(yaml: data, .default, yamsConstructor),
       let result = yaml as? [String: Any] else {
         fatalError("Unable to load fixture content")
     }
 
     return result
   }
+
+  private static let yamsConstructor: Constructor = {
+    var scalarMap = Constructor.defaultScalarMap
+    scalarMap[.bool] = Fixtures.constructBool
+    return Constructor(scalarMap)
+  }()
+
+  // swiftlint:disable:next discouraged_optional_boolean
+  private static func constructBool(from scalar: Node.Scalar) -> Bool? {
+    switch scalar.string.lowercased() {
+    case "true":
+      return true
+    case "false":
+      return false
+    default:
+      return nil
+    }
```

Comment:
Uhm, I'm not sure if that'd be what the user would expect? We say we use Yams, maybe they expect the default behaviour?

I was forced to do this, because in some contexts (I think the strings ones), we have a string value of "no" that needs to remain an actual `"no"` string 😅

Reply:
According to the YAML specs, it is expected behaviour:
http://yaml.org/type/bool.html

Reply:
Yeah I mean in the generated code sorry.
What I meant was:

1) If the constructor you've implemented here does follow the YAML specs then why isn't that the default constructor used by Yams? Maybe worth a PR in Yams?
2) If you have to do that here shouldn't we have to do the same in our own code when using Yams to parse the config file?

_(Maybe I misunderstand the whole thing, on my phone here so hard to browse the existing code or specs)_

Reply:
Ah wait ok I think I understand now.
 - The YAML spec allows true, false, no, yes, y, … to all be parsed as booleans
 - Yams follows that spec
 - You had to workaround that and not follow the spec in the test because you're using a YAML file in your fixtures that has "no" as a value and you expect it to be parsed as a string?

isn't YAML (and thus Yams) supposed to distinguish `yes` (boolean)  from `"yes"` (forced as string, because quoted)?

Reply:
YAML strings don't necessarily have quotes around them 🤷‍♂️ 
We can set a flag in Yams when writing to file to add type information, but it adds it **everywhere**, making the contexts unreadable and really verbose.

Reply:
I know it's not mandatory to have quotes around strings in YAML. But if you do have quotes then it's always parsed as a string, which allows to represent strings like `"123"` without being parsed into numbers, or strings like `"no"` without being parsed as a boolean when reading the file

 According to the YAML specs, when you write a YAML file, if the string isn't ambiguous (neither looks like a number or a boolean literal) then indeed you can omit quotes, and most people do. But if it's ambiguous, using quotes helps disambiguate and ensure its parsed as string by any YAML-compliant parser.

Reply:
What I mean here is that it's the responsibility of the person writing the YAML file to be careful to wrap the `no` into quotes on their YAML file if they want it to be parsed (by any spec-compliant YAML parser, Yams or otherwise) as string and not boolean.
It shouldn't be the responsibility of the parser to alter the specs to allow parsing unquoted `no` literals as strings (instead of boolean like the spec described)